test/base/src/androidTest/kotlin/io/realm/test/shared/SampleTests.kt

```diff
@@ -257,4 +259,47 @@ class SampleTests {
                 assertEquals(1, objects.size)
             }
     }
+
+    // Exhaustive test for all types are done in RealmListTest.assignField to leverage on the
+    // RealmListTest infrastructure
+    // @Test
+    // fun list_assign_allTypes() {}
+
+    @Test
+    fun list_assign_unmanaged() {
+        realm.writeBlocking {
+            val objectList: List<Sample> = (0..9).map { Sample().apply { intField = it } }
+            val sample = copyToRealm(Sample()).apply { objectListField = objectList.toRealmList() }
+        }
+        assertEquals(11, realm.query<Sample>().count().find())
```

Comment:
Ahh, good old inclusive `..` in both ends 🙄

Reply:
Yeah. _All inclusive_ seems to be the new standard for next generations 👴